requirements.txt

```diff
@@ -5,6 +5,7 @@ ipywidgets>=7.5
 xgboost>=0.82,<1.3.0
 catboost>=0.20
 lightgbm>=2.3.1,<3.1.0
+MarkupSafe==1.1.1
```

Comment:
@bchen1116 why add this to our requirements? Shouldn't this only be in our docs requirements?

Reply:
When I just added this to the doc requirements, we still had failing tests. I'm not sure why it happened, but adding it to requirements solved the issue

Reply:
RTD installs the docs requirements then evalml. evalml installs `MarkUpSafe` via `ipywidgets` so if we pin markupsafe in docs requirements, it'll still be overwritten when RTD installs evalml

Reply:
@freddyaboulton understood

[Here](https://github.com/alteryx/evalml/blob/main/.readthedocs.yaml#L21) is where we configure what gets installed in our RTD builds. I wonder if we now need another requirements file, which gets installed _after_ evalml is installed, and we can move the markupsafe pin there. I'm trying that out now in a draft PR #2261